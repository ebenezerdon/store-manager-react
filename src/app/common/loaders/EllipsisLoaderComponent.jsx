import React from 'react';
import { ellipsisLoader } from './svg';

const EllipsisLoaderComponent = () => {
  return <img className="ellipsis" src={ellipsisLoader} alt="loader" />;
};

export default EllipsisLoaderComponent;
