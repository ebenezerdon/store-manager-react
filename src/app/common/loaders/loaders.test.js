import React from 'react';
import { shallow } from 'enzyme';
import EllipsisLoaderComponent from './EllipsisLoaderComponent';

describe('It should render the EllipsisLoader', () => {
  it('should render the EllipsisLoaderComponent', () => {
    const component = shallow(<EllipsisLoaderComponent />);
    expect(component.exists()).toBe(true);
    expect(component).toMatchSnapshot();
  });
});
