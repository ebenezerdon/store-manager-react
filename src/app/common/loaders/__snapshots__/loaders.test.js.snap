// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render the EllipsisLoader should render the EllipsisLoaderComponent 1`] = `
<img
  alt="loader"
  className="ellipsis"
  src={Object {}}
/>
`;
